vita3k/gui/src/gui.cpp

```diff
@@ -63,7 +63,7 @@ void draw_info_message(GuiState &gui, EmuEnvState &emuenv) {
         ImGui::SetNextWindowPos(ImVec2(emuenv.viewport_pos.x + (display_size.x / 2) - (WINDOW_SIZE.x / 2.f), emuenv.viewport_pos.y + (display_size.y / 2.f) - (WINDOW_SIZE.y / 2.f)), ImGuiCond_Always);
         ImGui::PushStyleVar(ImGuiStyleVar_ChildRounding, 10.f * SCALE.x);
         ImGui::BeginChild("##info", WINDOW_SIZE, true, ImGuiWindowFlags_NoSavedSettings | ImGuiWindowFlags_NoDecoration);
-        auto title = fmt::format("{}", spdlog::level::to_string_view(gui.info_message.level));
+        auto title = fmt::format(fmt::runtime(gui.info_message.title), spdlog::level::to_string_view(gui.info_message.level));
         title[0] = std::toupper(title[0]);
```

Comment:
```suggestion
        auto title = fmt::format("{}", gui.info_message.title);
```
for this, after more look your code, you can remove level, sorry again
You have good works for can set all langue for error and info 
so you can just keep only like i suggest without std::toupper after
level value just keep used for log only so